lib/database/achievement.php

```diff
@@ -597,11 +600,28 @@ function UploadNewAchievement(
                         );
                     }
                 } else {
+                    $stringArray = [];
+
+                    if ($changingPoints) {
+                        array_push($stringArray, "points");
+                    }
+                    if ($changingBadge) {
+                        array_push($stringArray, "badge");
+                    }
+                    if ($changingWording) {
+                        array_push($stringArray, "wording");
+                    }
+                    if ($changingLogic) {
+                        array_push($stringArray, "logic");
+                    }
+
+                    $editString = implode(', ', $stringArray);
+
                     addArticleComment(
                         "Server",
                         \RA\ArticleType::Achievement,
                         $idInOut,
-                        "$author edited this achievement.",
+                        "$author edited this achievements $editString.",
```

Comment:
possessive form should be "achievement's".

Reply:
Thanks, should be correct now